cf/rds.yaml

```diff
@@ -1,4 +1,5 @@
 AWSTemplateFormatVersion: "2010-09-09"
+Transform: AWS::SecretsManager-2020-07-23
```

Comment:
What does this `Transform` do?

Reply:
`Transform` allows AWS to inject CF resources on our behalf. [In our case](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/transform-aws-secretsmanager.html), this is required to use the AWS-provided [secret rotation lambda](https://docs.aws.amazon.com/secretsmanager/latest/userguide/reference_available-rotation-templates.html#sar-template-postgre-singleuser).